webextensions/api/scripting.json

```diff
@@ -165,6 +165,102 @@
               "safari_ios": "mirror"
             }
           },
+          "InjectionResult": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "88"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": "102"
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "15.4"
+                },
+                "safari_ios": "mirror"
+              }
+            },
+            "documentId": {
+              "__compat": {
+                "support": {
+                  "chrome": {
+                    "version_added": "106"
+                  },
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": "102"
+                  },
+                  "firefox_android": "mirror",
+                  "opera": "mirror",
+                  "safari": {
+                    "version_added": "15.4"
+                  },
+                  "safari_ios": "mirror"
+                }
+              }
+            },
+            "error": {
+              "__compat": {
+                "support": {
+                  "chrome": {
+                    "version_added": false,
+                    "notes": "See <a href='https://crbug.com/1271527'>Issue 1271527: Propagate errors from scripting.executeScript to InjectionResult</a>."
+                  },
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": "102"
+                  },
+                  "firefox_android": "mirror",
+                  "opera": "mirror",
+                  "safari": {
+                    "version_added": "15.4"
```

Comment:
@xeenon Could you confirm whether this is accurate? Does Safari propagate the error in the scripting.executeScript API, and if yes, since when?

Reply:
@xeenon are you able to review this? I would like to get this PR merged?